codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketServerProtocolHandshakeHandler.java

```diff
@@ -110,6 +105,10 @@ public void operationComplete(ChannelFuture future) throws Exception {
         }
     }
 
+    private boolean isNotWebSocketPath(FullHttpRequest req) {
+        return checkStartsWith ? !req.uri().startsWith(websocketPath) : !req.uri().equals(websocketPath);
```

Comment:
@doom369 cool. didn't know what concat optimization can stumble on the such thing..

> thus concat operation became available for OptimizeStringConcat optimization